doc/admin-guide/files/records.yaml.en.rst

```diff
@@ -2488,6 +2488,29 @@ Cache Control
    used from the asynchronous IO threads when IO finishes and the ``CacheVC`` lock or stripe lock is
    required.
 
+.. ts::cv:: CONFIG proxy.config.cache.max_disk_errors INT 5
```

Comment:
Interestingly, these don't render:
https://ci.trafficserver.apache.org/job/Github_Builds/job/docs/4133/artifact/output/10960/docbuild/html/admin-guide/files/records.yaml.en.html#ram-cache

I think the issue is `ts:cv::` is correct, but `ts::cv::` (note the two colons after `ts` instead of one) is incorrect. It looks like you're the victim of copying and pasting from a previous mistake. And I'm glad you are so we can fix it.

Can you please search for all `ts::cv::` in this file and replace it with the correct `ts:cv::`? I think I count four such mistakes (including the two in this patch).

Reply:
Thanks for the catch!  I've fixed the offending `ts::cv::` lines.